homework3/jenkins.postman_collection.json

```diff
@@ -0,0 +1,335 @@
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "https://tr.a1qa.com/index.php?/api/v2/get_suites/{{TRprojectID}}",
+					"protocol": "https",
+					"host": [
+						"tr",
+						"a1qa",
+						"com"
+					],
+					"path": [
+						"index.php"
+					],
+					"query": [
+						{
+							"key": "/api/v2/get_suites/{{TRprojectID}}",
+							"value": null
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "section",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "eb729513-595c-4728-bac1-893e84d7dd9d",
+						"exec": [
+							"pm.test(\"Test suite is created\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							"\r",
+							"const Body = pm.response.json();\r",
+							"\r",
+							"pm.collectionVariables.set(\"SectionID\", Body.id);"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n\t\"suite_id\": {{SuiteID}},\r\n\t\"name\": \"Postman_Advanced\"\r\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "https://tr.a1qa.com/index.php?/api/v2/add_section/{{TRprojectID}}",
+					"protocol": "https",
+					"host": [
+						"tr",
+						"a1qa",
+						"com"
+					],
+					"path": [
+						"index.php"
+					],
+					"query": [
+						{
+							"key": "/api/v2/add_section/{{TRprojectID}}",
+							"value": null
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "get from JIRA",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "0f80dcf5-f191-46e5-a4b9-20f18ccfef9c",
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"id": "4d3e91c0-1117-43f3-98d3-5e9b51215912",
+						"exec": [
+							"const date = require (\"moment\");\r",
+							"const body = pm.response.json();\r",
+							"const CaseArray = [];\r",
+							"const TempTRResponceBody = [];\r",
+							"pm.collectionVariables.set(\"JiraProjectName\", body.issues[0].fields.project.key);\r",
+							"\r",
+							"const GetTRCases = {\r",
+							"    url: `https://tr.a1qa.com/index.php?/api/v2/get_cases/${pm.collectionVariables.get(\"TRprojectID\")}&suite_id=${pm.collectionVariables.get(\"SuiteID\")}`,\r",
+							"    method: 'GET',\r",
+							"    header: {\r",
+							"        'Content-Type': 'application/json',\r",
+							"        'Authorization': request.headers.authorization\r",
+							"          },\r",
+							"};\r",
+							"pm.sendRequest(GetTRCases, function (err, res) {\r",
+							"    console.log(err ? err : \"Get TestRail issues - OK\");\r",
+							"    res.json().forEach(index => TempTRResponceBody.push(index.title));\r",
+							"});\r",
+							"\r",
+							"CaseArray.forEach(TestCase => {\r",
```

Comment:
`CaseArray` is always empty array, which means that forEach loop will never start